swagger.js

```diff
@@ -32,12 +32,64 @@ const doc = {
         },
       },
     },
-    TokenizedUser: {
-      user: {
-        name: 'John Doe',
-        userId: '64ef5885c2a489217c571253',
-        role: { '@enum': ['user', 'admin'] },
+
+    Order: {
+      $_id: 'idabc123456',
+      $user: {
+        $id: 'idabc123456',
+        $name: 'John Doe',
+        $username: 'johndoe',
+        $email: 'john@example.com',
+        $role: { '@enum': ['user', 'admin'] },
+      },
+
+      $orderStatus: {
+        '@enum': [
+          'pending',
+          'payment_successful',
+          'payment_failed',
+          'cancelled',
+          'complete',
+        ],
+        default: 'pending',
       },
+      orderItems: [
+        {
+          $album: '5f7f1f7a5c3f2b2d9c1b0b9d',
+          $quantity: 2,
+          $_id: 'idabc12345',
+        },
+      ],
+      $subtotal: 9.99,
+      $tax: 0.75,
+      $total: 10.74,
+      $createdAt: '2023-08-08T20:45:00.942Z',
+      $updatedAt: '2023-08-08T20:45:00.942Z',
+      paymentIntentId: 'pi_abc123456',
+    },
+
+    NewOrder: {
+      $subtotal: 100,
+      $tax: 0.15,
+      $total: 115,
+      $orderItems: [{ album: 'abcdef12345', quantity: 2 }],
+    },
+
+    OrderList: {
+      orders: [
+        {
+          user: 'Jane Smith',
```

Comment:
on the `OrderList` used in the `GET /orders` endpoint, the `user` field is not populated so we actually just get back the user's id not their name. Here's an example response:

```json
{
  "orders": [
    {
      "_id": "64faeb31e00e8a4c21dfec94",
      "user": "64d44bc8337399ccf9ad7e4d",
      "orderStatus": "complete",
      "tax": 0.4,
      "subtotal": 100,
      "total": 140,
      "orderItems": [
        {
          "album": "64d36bd25c5dec8e599bee05",
          "quantity": 1,
          "_id": "64faeb31e00e8a4c21dfec95"
        },
        {
          "album": "64d2ae2308a725b72bd5c0da",
          "quantity": 4,
          "_id": "64faeb31e00e8a4c21dfec96"
        },
        {
          "album": "64d2ae2308a725b72bd5c0db",
          "quantity": 2,
          "_id": "64faeb31e00e8a4c21dfec97"
        }
      ],
      "createdAt": "2023-09-08T09:36:49.933Z",
      "updatedAt": "2023-09-08T09:36:49.933Z",
      "__v": 0
    },
    {
      "_id": "64fc39e8d4e3a9f350fa6b81",
      "user": "64d44bc8337399ccf9ad7e4d",
      "orderStatus": "complete",
      "tax": 0.4,
      "subtotal": 100,
      "total": 140,
      "orderItems": [
        {
          "album": "64d36bd25c5dec8e599bee05",
          "quantity": 1,
          "_id": "64fc39e8d4e3a9f350fa6b82"
        },
        {
          "album": "64d2ae2308a725b72bd5c0da",
          "quantity": 4,
          "_id": "64fc39e8d4e3a9f350fa6b83"
        },
        {
          "album": "64d2ae2308a725b72bd5c0db",
          "quantity": 2,
          "_id": "64fc39e8d4e3a9f350fa6b84"
        }
      ],
      "createdAt": "2023-09-09T09:24:56.384Z",
      "updatedAt": "2023-09-09T09:24:56.384Z",
      "__v": 0
    },
    {
      "_id": "64fc3a61becbd34df27e3b59",
      "user": "64d44bc8337399ccf9ad7e4d",
      "orderStatus": "complete",
      "tax": 0.4,
      "subtotal": 100,
      "total": 140,
      "orderItems": [
        {
          "album": "64d36bd25c5dec8e599bee05",
          "quantity": 8,
          "_id": "64fc3a61becbd34df27e3b5a"
        },
        {
          "album": "64d2ae2308a725b72bd5c0da",
          "quantity": 84,
          "_id": "64fc3a61becbd34df27e3b5b"
        },
        {
          "album": "64d2ae2308a725b72bd5c0db",
          "quantity": 82,
          "_id": "64fc3a61becbd34df27e3b5c"
        }
      ],
      "createdAt": "2023-09-09T09:26:57.926Z",
      "updatedAt": "2023-09-09T09:26:57.926Z",
      "__v": 0
    },
    {
      "_id": "64fcdc35d7349bd5eff622e2",
      "user": "64ef5e90086e1c19b888c72c",
      "orderStatus": "payment_successful",
      "tax": 0.15,
      "subtotal": 500,
      "total": 550,
      "orderItems": [
        {
          "album": "64d36b781a3159bca4e17193",
          "quantity": 5,
          "_id": "64fcdc35d7349bd5eff622e3"
        }
      ],
      "createdAt": "2023-09-09T20:57:25.668Z",
      "updatedAt": "2023-09-09T20:57:33.322Z",
      "__v": 0,
      "paymentIntentId": "pi_3NoYDaKnb0YvaiB31RGQGLwz"
    },
    {
      "_id": "64fcdc761543ae4a89fcaa52",
      "user": "64ef5e90086e1c19b888c72c",
      "orderStatus": "payment_successful",
      "tax": 0.15,
      "subtotal": 500,
      "total": 550,
      "orderItems": [
        {
          "album": "64d36b781a3159bca4e17193",
          "quantity": 5,
          "_id": "64fcdc761543ae4a89fcaa53"
        }
      ],
      "createdAt": "2023-09-09T20:58:30.889Z",
      "updatedAt": "2023-09-09T20:58:37.257Z",
      "__v": 0,
      "paymentIntentId": "pi_3NoYEdKnb0YvaiB31bZI5Twx"
    },
    {
      "_id": "64fcdc8f4331a018d0c5aa3b",
      "user": "64ef5e90086e1c19b888c72c",
      "orderStatus": "payment_successful",
      "tax": 0.15,
      "subtotal": 500,
      "total": 550,
      "orderItems": [
        {
          "album": "64d36b781a3159bca4e17193",
          "quantity": 5,
          "_id": "64fcdc8f4331a018d0c5aa3c"
        }
      ],
      "createdAt": "2023-09-09T20:58:55.443Z",
      "updatedAt": "2023-09-09T21:03:33.139Z",
      "__v": 0,
      "paymentIntentId": "pi_3NoYF2Knb0YvaiB314UGm5o1"
    },
    {
      "_id": "64fcddc6d3ab1a4d7a54ac1c",
      "user": "64ef5e90086e1c19b888c72c",
      "orderStatus": "payment_successful",
      "tax": 0.15,
      "subtotal": 500,
      "total": 550,
      "orderItems": [
        {
          "album": "64d36b781a3159bca4e17193",
          "quantity": 5,
          "_id": "64fcddc6d3ab1a4d7a54ac1d"
        }
      ],
      "createdAt": "2023-09-09T21:04:06.154Z",
      "updatedAt": "2023-09-09T21:16:15.528Z",
      "__v": 0,
      "paymentIntentId": "pi_3NoYK2Knb0YvaiB30dlLP6TG"
    }
  ],
  "count": 7
}
```

so weshould update the definition here to match that:

```suggestion
          user: '64d44bc8337399ccf9ad7e4d',
```